.github/workflows/promptflow-release-testing-matrix.yml

```diff
@@ -105,7 +105,7 @@ jobs:
       matrix:
         os: [windows-latest]
         testType: [sdk-cli, executor-e2e, executor-unit]
-        pythonVersion: ['3.9', '3.10', '3.11']
+        pythonVersion: ['3.8', '3.9', '3.10', '3.11', '3.12']
```

Comment:
No need to cover 3.12?
![image](https://github.com/microsoft/promptflow/assets/7776147/f5f78a0a-47ec-489f-a47e-d2763ba7904c)

Reply:
Not a blocking pipeline